src/pages/api/repositories/columnRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { ColumnModel } from "../database/cassandra";
+import { BaseRepository } from "./baseRepository";
+
+class ColumnRepository extends BaseRepository<typeof ColumnModel> {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.